src/components/IOUConfirmationList.js

```diff
@@ -416,6 +414,7 @@ IOUConfirmationList.defaultProps = defaultProps;
 export default compose(
     withLocalize,
     withWindowDimensions,
+    withNetwork(),
```

Comment:
Why are you calling the method, instead of a reference to the method? like withLocalize

Reply:
This HOC was created using this factory method here -> https://github.com/Expensify/App/blob/f5bb9c8af9b38ccfc78a31e98844f5689fe56b42/src/components/createOnyxContext.js#L29

Which allows us to pass an optional `propName` and function to modify the value before passed to the component.